pkg/cmd/extension/push/cmd.go

```diff
@@ -18,7 +18,7 @@ import (
 	"errors"
 	"fmt"
 
-	"github.com/docker/distribution/reference"
+	"github.com/containerd/containerd/reference/docker"
```

Comment:
we don't need to import the docker distribution to verify a reference. NOTE: we still import it indirectly due to oras, but I can manage that in a follow-up PR